benchmarks/torchao/benchmark.py

```diff
@@ -0,0 +1,308 @@
+import argparse
```

Comment:
Is there a strong reason that we need to duplicate this folder? Can we just refer the original one?

Reply:
what do you mean by original one?

Reply:
Sounds good, I can use gpt-fast